clients/cobol-lsp-vscode-extension/src/type/e4eApi.d.ts

```diff
@@ -139,11 +139,10 @@ type E4EExternalConfigurationResponse = {
 interface e4eResponse {
   configuration: {
     name: string;
-    libs: any[];
+    libs: [EndevorElement | EndevorMember];
   };
   api: E4E;
   profile: ResolvedProfile;
   uri: string;
-  elements: EndevorElement[];
-  members: EndevorMember[];
+  elements: [EndevorElement | EndevorMember];
```

Comment:
These should be arrays, not one-element tuples...

Reply:
it is not [aaa , aaa ] but a [aaa | bbb]. So it is an array of objects consisting either EndevorElement or EndevorMember

Reply:
It is actually supposed to be `(EndevorElement | EndevorMember)[]`